backend/main/src/Services/GetUserIdService/GetUserService.ts

```diff
@@ -17,11 +17,12 @@ export class GetUserService implements IGetUserService {
 		}
 		// get token from bearer token
 		const authToken = authHeader.split(" ")[1];
-		const decodedToken = jwt.decode(authToken);
+		// biome-ignore lint/suspicious/noExplicitAny: allow any. This is specified by the Cognito.
+		const decodedToken: any = jwt.decode(authToken);
 		if (!decodedToken)
 			return Err(new InvalidTokenError("The token is invalid."));
 
```

Comment:
Would you be willing to add the type-checking? The reason that you want to assign any to the variable might be an IDE error. That error comes from the decodedToken, which can be a string or a JwtPayload. By adding this check, an IDR can infer the variable is the JwtPayload.

```suggestion
		if (typeof decodedToken === "string")
			return Err(new InvalidTokenError("The token is invalid."));

```

Reply:
Thank you for pointing it out! I updated it

Reply:
Oh, I mean that by adding this check, we can remove the 'any' annotation and the biome-ignore.

Reply:
Omg, I didn't notice the solution. I got it. I didn't realize JwtPayload type has `[key: string]`. I just thought the type didn't have an email property.  Thank you so much! I fixed it.